src/Views/Games/2048/Instructions.js

```diff
@@ -5,7 +5,7 @@ import clsx from "clsx";
 
 const useStyles = makeStyles(theme => ({
     root: {
-
+    	whiteSpace: 'pre-line'
```

Comment:
Let root take up all of it's available space and using padding, margins, or flexboxes to handle spacing